### DISCUSSION TITLE
Howdy Y'all!

### DISCUSSION THREAD

**Discussion by amandagrays** [OpenMined/courses#32] [Introduce Yourself]
Amanda here from San Diego, California! I've been sheltering-in-place with my cat Amelia since March 2019.

I lost my job twice last year and am starting a short term role next week providing remote IT support to federal COVID-19 contact tracers.

My brief periods of being unemployed afforded me the time to explore and expand my knowledge in Cloud Environments, AI, Data Science, digital identities, and consumer privacy. I love to learn and I think that is why I ended up here.

My ties to AI and Privacy are mostly from personal experience as a consumer traumatized by learning the truth about what was being done with my data this year and realizing I needed to monitor my digital identity and protect her from a great many things.

The state I live in established a consumer rights act (CCPA) this year with special emphasis on being entitled transparency on what personal data is being gathered, how it is used, who it is sold to, and consent. In July of this year that act became active and I took an alice in wonderland dive into meeting my digital clones and learning how they were interacting with anyone with access to my data. Long story short I spent a lot of time taking my digital Identity back and feel a groups like this are absolutely necessary for the direction technology is heading. The generations alive today have a responsibility to make sure privacy standards are established, well known, and respected.

I didn't think big data was going to take responsibility. Seeing Facebook in the list of contributing companies was a shock. I am very interested to see how things pan out. Some other companies established their own alliance and proclaimed that alliance to be the leading authority on insuring privacy standards are established and standardized. I wonder if this group will be working with them to some degree?

Anyway, receiving an invitation to this movement was a surprise and very exciting. Things went quiet for a while after I was invited and I thought I imagined it. I'm happy to see organized content and other like minded folks actually manifested.

Nice to meet you all I look forward to learning with and from you all!

![IMG_20200906_183258](https://user-images.githubusercontent.com/76879630/103463335-78eb3d80-4ce0-11eb-9bd8-ee0b8b8a42dd.jpg)
![PXL_20201123_223519532 MP~2](https://user-images.githubusercontent.com/76879630/103463338-7b4d9780-4ce0-11eb-8555-5fbbb02101dd.jpg)

-Amanda

**Comment by yemikifouly** [comment]
Hello Amanda! 🙂 It is very nice to meet you! I think you will like exchanging with a lot of people in our Slack community because most of us are just like you: driven by the need for more privacy and control over how our - and everyone's - information flows. If you watch the video you will see that Andrew, OpenMined's leader, is extremely passionate about the topic. His enthusiasm and passion almost jump out the screen. 🙂 I don't know that we have definite perfect solutions but I think we can definitely provide you with a lot more information and share some technologies that can help solve the issues that concern you and us.  🙂 

I am really glad you decided to sign up for the courses and I hope you enjoy the first one very much. I suspect you will find it quite interesting.  🙂
